File: docs/components_page/components/button_group/outline.py
import dash_bootstrap_components as dbc

button_group = dbc.ButtonGroup(
    [
        dbc.Button("Left", outline=True, color="primary"),
        dbc.Button("Middle", outline=True, color="primary"),
        dbc.Button("Right", outline=True, color="primary"),
    ]
)

File: docs/components_page/components/button_group/outline.R
library(dashBootstrapComponents)

button_group <- dbcButtonGroup(
  list(
    dbcButton("Left", outline = TRUE, color = "primary"),
    dbcButton("Middle", outline = TRUE, color = "primary"),
    dbcButton("Right", outline = TRUE, color = "primary")
  )
)
